changelogs/unreleased/5439-desired-state-fix.yml

```diff
@@ -0,0 +1,3 @@
+description: "Fix resolving freezing of compliance check page"
```

Comment:
can you add the missing section ? Otherwise there won't be any description present in the changelogs when released :)